source/extensions/retry/priority/other_priority/other_priority.cc

```diff
@@ -84,6 +71,22 @@ void OtherPriorityRetryPriority::adjustForAttemptedPriorities() {
     }
   }
 }
+
+std::pair<std::vector<uint32_t>, uint32_t> OtherPriorityRetryPriority::adjustedHealth() const {
```

Comment:
Can we get your comment back?

// create an adjusted health view of the priorities, where attempted priorities are
// given a zero weight.